src/parse/error.rs

```diff
@@ -78,6 +80,13 @@ impl LexicalError {
                 "I can't figure out what to do with this character.",
                 vec!["Hint: Is it a typo?".to_string()],
             ),
+	        LexicalErrorType::CamelCaseName { name } => (
+		        "This is not a valid name.",
+		        vec![
+			        "Hint: In Gleam names must start with a lowercase letter and contain only lowercase letters, numbers, and '_'.".to_string(),
```

Comment:
Can we wrap this message onto two lines? Around 70 chars

```suggestion
			        "Hint: In Gleam names must start with a lowercase letter and contain".to_string(),
			        "only lowercase letters, numbers, and '_'.".to_string(),
```

Reply:
I'm unsure if it's still too long for you because I shortened the message per @\itsgreggreg's request. Let me know if you still think it should be shortened!